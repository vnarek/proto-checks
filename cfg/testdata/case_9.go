package main

func main() {
	x := new(int)
	max := new(int)
	*max = 5
	for i := *x; i < *max; i++ {
		x = &i
	}
	x = nil
}
