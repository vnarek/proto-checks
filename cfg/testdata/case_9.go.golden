[x = alloc]
[max = alloc]
[i = *x]
[x = &i]
[x = null]
